content/20.treatments.md

```diff
@@ -558,24 +489,22 @@ One trial is currently investigating the utility of vitamin D as an immune-modul
 Other trials are also underway examining various factors including mortality, symptom recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein (CRP) and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration [@clinicaltrials:NCT04385940; @clinicaltrials:NCT04334005; @clinicaltrials:NCT04411446; @clinicaltrials:NCT04344041].
 Concomitant administration of vitamin D with pharmaceuticals such as aspirin [@clinicaltrials:NCT04363840] and bioactive molecules such as resveratrol [@clinicaltrials:NCT04400890] are also under investigation.
 
-**Summary.**
-Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin D status warrants further investigation.
-This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
-This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection.
+Whether or not vitamin D status is linked to COVID-19 outcomes, even spuriously, thus remains open for debate.
+Once again, supplementation of vitamin D and maintaining a healthy diet for optimum vitamin D status is unlikely to carry major health risks while the possible link between vitamin D status and COVID-19 is investigated.
+However, pursuing to elevate vitamin D levels through sunlight exposure does carry additional risks, as many densely populated cities around the world are utilizing ‘stay in place’ orders to enforce social distancing guidelines.
+Given the lack of conclusive evidence in support of vitamin D, it is not clear that these guidelines present additional risk.
+However, to the extent that people are able to maintain safe exposure to sunlight, there is a possibility that it could improve endogenous synthesis of vitamin D, potentially strengthening the immune system.
```

Comment:
It can be got in diet too, maybe acknowledge that too if you think its relevant, I didn't want to mess up your nice wording here :)